src/Gym/Envs/IEnv.cs

```diff
@@ -14,7 +16,7 @@ public interface IEnv {
         NDArray Reset();
         Step Step(int action);
         Task<Step> StepAsync(int action);
-        byte[] Render(string mode = "human");
+        Image Render(string mode = "human");
```

Comment:
I think it is more right to use `Image` which is parent class of `Image<TRGB>` because not all renderings might have alpha in them.

What are your thoughts on this regard?
Also, this change has broken your example projects.

Reply:
Yeah i assumed all renderings will have alpha but that comes from my limited experience with the Gym envs. 
Maybe i can still use Image, if it's not possible will simply downcast the image in the example and throw NotSupported if it fails.

Reply:
Yes the example can use Image. Pushed the update